content/doc/developer/views/index.adoc

```diff
@@ -35,6 +35,9 @@ Jelly will emit XML comments that help understand Jelly view composition from fr
 This can help understand how a view is composed.
 The corresponding Java system property is link:/doc/book/managing/system-properties/#stapler-jelly-trace[`stapler.jelly.trace`].
 
+NOTE: Support for Jelly tracing is broken in Jenkins 2.287 and earlier ([JENKINS-65288](https://issues.jenkins.io/browse/JENKINS-65288)).
```

Comment:
⁉️  It seems the previous (at least) does not like the ([]()) format for link

![Screenshot_2022-10-07_094949_001](https://user-images.githubusercontent.com/2662497/194501078-2105b08d-9f53-4cbd-9ca9-ae0086fc8b2f.png)

Reply:
Oops, accidental Markdown.
```suggestion
NOTE: Support for Jelly tracing is broken in Jenkins 2.287 and earlier (https://issues.jenkins.io/browse/JENKINS-65288[JENKINS-65288]).
```